mindmeld/text_preparation/tokenizers.py

```diff
@@ -241,8 +241,9 @@ def __init__(self, language, spacy_model_size="sm"):
             spacy_model_size (str, optional): Size of the Spacy model to use. ("sm", "md", or "lg")
         """
         self.spacy_model = SpacyModelFactory.get_spacy_language_model(
-            language, spacy_model_size
+            language, spacy_model_size, disable=["tagger", "parser", "ner"]
```

Comment:
Seems like we can also disable `lemmatizer` and `attribute_ruler`

Reply:
Sure, expanded the disable list